ignite/contrib/handlers/base_logger.py

```diff
@@ -41,7 +40,13 @@ class BaseOutputHandler(BaseHandler):
     Helper handler to log engine's output and/or metrics
     """
 
-    def __init__(self, tag, metric_names=None, output_transform=None, global_step_transform=None):
+    def __init__(
+        self,
+        tag,
```

Comment:
You missed a type annotation here, probably `tag: str`

Reply:
ah, sorry. Done.